src/main/java/jp/ac/keio/bio/fun/xitosbml/image/SpatialImage.java

```diff
@@ -322,15 +336,28 @@ public void setAdjacentsList(ArrayList<ArrayList<String>> adjacentsList) {
 	 * @param path the path to the directory
 	 * @param name the name of TIFF file
 	 */
-	public void saveAsImage(String path, String name){
+	public void saveAsImage(String path, String name) {
 		FileSaver fs = new FileSaver(img);
-		if(name == null) return;
-		if(depth > 1)
+		if (name == null)
+			return;
+		if (depth > 1)
 			fs.saveAsTiffStack(path + "/" + name + ".tiff");
 		else
 			fs.saveAsTiff(path + "/" + name + ".tiff");
 	}
 
+	/**
+	 * Save image as TIFF file --> for the CUI version
+	 * 
+	 * @param path2
+	 */
+	public void saveAsImageCui(String path2) {
+		FileSaver fs = new FileSaver(img);
+		if (path2 == null)
+			return;
+		fs.saveAsTiff(path2 + "_sptl" + ".tiff");
+	}
+
```

Comment:
Maybe just overload the method? So we can just have a method named `saveAsImage` running with either 1 argument or 2.

Reply:
Done. Please refer to https://github.com/Medha-B/XitoSBML/issues/5.